### PR TITLE
[NO-CI] Make fetch available on the window for tests

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

**Review by deomsj — APPROVED** [review]
Nice. thanks for the investigation and quick fix!

image

File: spec/src/constructorio.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ describe(`ConstructorIO${bundledDescriptionSuffix}`, () => {
 
     if (bundled) {
       ConstructorIO = window.ConstructorioClient;
+
+      // Make global fetch available on the window
+      window.fetch = fetch;
```
Comment (jjl014) [inline spec/src/constructorio.js:34]: I think we might be able to move this to the `jsdom-global.js` utils file so we don't need to specify it in all these files.

**Review by jjl014 — APPROVED** [review]
LGTM! 🚀 

Thanks for working on the fix! I just left one comment which might help clean things up a bit. 👍

File: spec/src/constructorio.js (hunk 1 above)
Comment (esezen) [reply spec/src/constructorio.js]: Fantastic idea! I forgot we have this file. Updating the code

**Pull request merged**